e2e/e2e_test.go

```diff
@@ -78,10 +84,30 @@ func Test_E2E(t *testing.T) {
 	transactionResponse, whbarBalanceBefore := verifyTransferToBridgeAccount(setupEnv, memo, whbarReceiverAddress, t)
 
 	// Step 2 - Verify the submitted topic messages
-	ethTransactionHash := verifyTopicMessages(setupEnv, transactionResponse, 1, t)
+	ethTransactionHash, receivedSignatures := verifyTopicMessages(setupEnv, transactionResponse, 1, t)
 
 	// Step 3 - Verify the Ethereum Transaction execution
 	verifyEthereumTXExecution(setupEnv, ethTransactionHash, whbarReceiverAddress, expectedWHbarAmount.Int64(), whbarBalanceBefore, t)
+
+	// Step 4 - Verify Transaction Record in the database
+	expectedTxRecord := verifyTransferRecordInDB(
+		setupEnv.Clients.RouterContract,
+		setupEnv.DBVerifiers,
+		transactionResponse.TransactionID,
+		"HBAR",
+		txFee,
+		gasPrice,
+		statusInfo{
+			status:          entity_transfer.StatusCompleted,
+			statusSignature: entity_transfer.StatusSignatureMined,
+			statusEthTx:     entity_transfer.StatusEthTxMined,
+			statusEthTxMsg:  entity_transfer.StatusEthTxMsgMined,
+		},
+		ethTransactionHash,
+		true, t)
+
+	// Step 5 - Verify Message Record in the database
+	verifyMessageRecordsInDB(setupEnv.DBVerifiers, expectedTxRecord, receivedSignatures, t)
```

Comment:
I think that it would be best if we make it cleaner and just do 1 step for both validations `verifyDatabaseRecords`